src/events.rs

```diff
@@ -20,9 +20,9 @@ pub enum EventType {
     #[serde(rename = "message_event")]
     Message { message: String },
     #[serde(rename = "success_event")]
-    Success { groups: String },
+    Success { group: String },
```

Comment:
Great catch @laurentsenta!

For the record: https://github.com/testground/sdk-go/blob/d78f61eccaa37b0d941f98650bce5dbb8fd7b7f1/runtime/runenv_events.go#L68-L70